src/main.rs

```diff
@@ -92,9 +92,19 @@ impl FromStr for LevelSizes {
     }
 }
 
-fn main() {
-    #[allow(deprecated)]
-    let matches = App::new(crate_name!())
+struct Config {
```

Comment:
I know that the code base doesn't really have them atm, but can you add doc comments to the new structs and functions to say what they do please?

Reply:
Ah sorry they're added in azren/documented_lib

Reply:
Woop!